deploy/ansible/deploy.yml

```diff
@@ -14,8 +14,14 @@
     - role: goshimmer-node
       nodeType: analysis-server
 
-- hosts: goshimmers
+- hosts: goshimmers:&internal_nodes
   vars:
     nodeType: goshimmer
   roles:
-    - goshimmer-node
\ No newline at end of file
+    - goshimmer-node
+
+- hosts: goshimmers:&public_nodes
+  vars:
+    nodeType: public-node
```

Comment:
I very much like this! So the folder under `/opt` will tell you what type of node you are dealing with.